packages/auto_mappr_annotation/lib/src/auto_mappr.dart

```diff
@@ -5,10 +5,10 @@ import 'package:auto_mappr_annotation/src/type_converter.dart';
 /// Annotates class which will be used as base for generated mappr.
 final class AutoMappr {
   /// List of mapprs.
-  final List<MapType<Object, Object>> mappers;
+  final List<MapType<Object?, Object?>> mappers;
```

Comment:
Not this :), mappers are ok to be no null since you mark the type as whole.

Reply:
Oh, whoops! 😳